CITUSDB_SETUP.md

```diff
@@ -107,13 +118,14 @@ with the same hostname that Django uses. Setting `CITUS_NODE_NAME`
 will override the host and port for master to worker communications.
 
 ## Setting up the database
+
 There are two ways to do the database setup. In both cases you will need
 to manually create the databases:
 
 ```
-$ psql -h localhost -p 5600 -U postgres -c 'create database commcare_ucr_citus;'
-$ psql -h localhost -p 5601 -U postgres -c 'create database commcare_ucr_citus;'
-$ psql -h localhost -p 5602 -U postgres -c 'create database commcare_ucr_citus;'
+$ psql -h localhost -p 5600 -U commcarehq -c 'create database commcare_ucr_citus;' postgres
```

Comment:
what does the `postgres` bit at the end do

Reply:
tells the command to be run on `postgres` db (else it defaults to `commcarehq` db which doesn't exist)